charts/pip-frontend/values.yaml

```diff
@@ -22,5 +22,11 @@ nodejs:
           alias: REDIS_PASSWORD
         - name: REDIS-PORT
           alias: REDIS_PORT
+        - name: AUTH-RETURN-URL
```

Comment:
AUTH_RETURN_URL will be an environment variable not a secret

Reply:
how do we set env variables? we need quick assistance